Bindings/Java/Matlab/tests/testOsimVec3.m

```diff
@@ -0,0 +1,28 @@
+%% clear working space
+clear all;close all;clc;
+%% import opensim libraries
+import org.opensim.modeling.*
+% Adding a call to Model(). If Model() is not instantiated, the test fails.
+m = Model();
+%% Define an abriatory an OpenSim Vec3 instance
+ov = Vec3(2,3,4);
```

Comment:
Probably fine, but there could, in theory, be some loss of precision as a result of these conversions. Might want to pick numbers like pi, -sqrt(2), inf.